internal/admin/manageUsers.go

```diff
@@ -0,0 +1,103 @@
+package admin
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"net/http"
+
+	"Twitter_like_application/internal/database/pg"
+	"Twitter_like_application/internal/services"
+
+	"github.com/gorilla/mux"
+)
+
+type UsersList struct {
+	ID       int    `json:"id"`
+	Username string `json:"username"`
+}
+
+type messageRequest struct {
+	UserID string `json:"id"`
+	Text   string `json:"message"`
+}
+
+func BlockUser(w http.ResponseWriter, r *http.Request) {
```

Comment:
this does not seem related to the " TWI-29 feat: add methods blocktweet,unblocktweet #43 "